### DISCUSSION TITLE
refresh API 404 not found

### DISCUSSION THREAD

**Discussion by jbalatero** [payloadcms/payload#319] [Q&A] [answered]
Hello, the [refresh](https://payloadcms.com/docs/authentication/operations#refresh) `/refresh` gives me HTTP 404 Not Found with body:

```
Cannot POST /api/[slug]/refresh
```

The request already contains header Authorization: JWT [token here]

Login and logout endpoints are working fine.

Currently using Payload 0.9.4

Thanks

**Comment by jmikrut** [comment, marked as answer]
Wow! Our docs are wrong. The REST endpoint is:

`POST` `/api/[:collectionSlug]/refresh-token`

We are going to fix this immediately. Thanks for catching this!

**Reply by jbalatero** [reply]
Thanks
